test/Table_Tests/src/Common_Table_Operations/Join/Cross_Join_Spec.enso

```diff
@@ -149,4 +151,12 @@ spec setup =
             r4 = [100, 4, 'b', 'd']
             r5 = [100, 4, 'a', 'x']
             expected_rows = [r0, r1, r2, r3, r4, r5]
-            r.should_equal expected_rows
+            case setup.is_database of
+                True -> r.should_contain_the_same_elements_as expected_rows
+                False -> r.should_equal expected_rows
+
+        if setup.is_database then
+            Test.specify "Cross join is not possible via call to .join" <|
+                t1 = table_builder [["X", [1, 2]], ["Y", [4, 5]]]
+                t2 = table_builder [["Z", ['a', 'b']], ["W", ['c', 'd']]]
+                Test.expect_panic_with (t1.join t2 join_kind=Join_Kind_Cross.Cross on=[]) Type_Error
```

Comment:
I guess this test does not need to be guarded with `is_database`.

Reply:
Done